cicd/pipelines/viewing-pipeline-logs-using-the-openshift-logging-operator.adoc

```diff
@@ -0,0 +1,35 @@
+[id='viewing-pipeline-logs-using-the-openshift-logging-operator']
+= Viewing pipeline logs using the OpenShift Logging Operator
+
+include::modules/common-attributes.adoc[]
+include::modules/pipelines-document-attributes.adoc[]
+
+
+:context: viewing-pipeline-logs-using-the-openshift-logging-operator
+
+
+toc::[]
+
+
+The logs generated by pipeline runs, task runs, and event listeners are stored in the respective pods. It is often useful to review and analyze such logs for troubleshooting and audits. However, retaining the pods indefinitely leads to unnecessary resource consumption and cluttered namespaces.
+
+To eliminate dependency on pods for viewing pipeline logs, you can use the OpenShift Elasticsearch Operator and the OpenShift Logging Operator provided by {pipelines-title}. These Operators helps you see pipeline logs using the link:https://www.elastic.co/guide/en/kibana/6.8/connect-to-elasticsearch.html[Elasticsearch Kibana] stack, even after deleting the pods containing the logs.
```

Comment:
```suggestion
To eliminate dependency on pods for viewing pipeline logs, you can use the OpenShift Elasticsearch Operator and the OpenShift Logging Operator provided by {pipelines-title}. These Operators help you to view pipeline logs by using the link:https://www.elastic.co/guide/en/kibana/6.8/connect-to-elasticsearch.html[Elasticsearch Kibana] stack, even after you have deleted the pods that contained the logs.
```

Reply:
Some questions for this one
1. Are there variables for those Operator names? Mostly to make your life easier when writing 🙂 we have a variable we use for the {ServerlessOperatorName} so I wasn't sure if there's something similar you want to use for things like OpenShift Logging Operator or maybe want to add these?
2. It says the Operators are provided by Pipelines, but are they really? Is there a dependency of installing some Pipelines component(s) before you can install these Operators, or should "provided by {pipelines-title}" maybe be removed?

Reply:
I did not find variables for these Operators, so keeping plaintext for now.